packages/smooth_app/lib/pages/preferences/user_preferences_contribute.dart

```diff
@@ -319,6 +335,19 @@ class UserPreferencesContribute extends AbstractUserPreferences {
       builder: (_) => tile,
     );
   }
+
+  Future<String> returnCountry(BuildContext context) async {
```

Comment:
Please remove that method, we don't need it.

Reply:
Whoops yeah thats my bad